packages/core/src/index.ts

```diff
@@ -57,6 +58,14 @@ export function buildSendMail(options: BuildSendMailOptions) {
   }
 
   return async function sendMail(mail: ComponentMail) {
+    // anonymous telemetry
+    if (anonymousId) {
```

Comment:
In the core package, there's no CLI argv to specify this so I'm gonna need to get it from somewhere else.  Am I gonna be able to read mailing.config.json from the project root or do I have to think of a different way to do this?